AnalyticsMessageStatusStorageQueueInbloundAdapter/function.json

```diff
@@ -0,0 +1,18 @@
+{
+    "bindings": [
+      {
+        "type": "queueTrigger",
+        "direction": "in",
+        "name": "messageviewfailure",
+        "queueName": "%MESSAGE_STATUS_FAILURE_QUEUE_NAME%",
+        "connection":"INTERNAL_STORAGE_CONNECTION_STRING"
+      }
+    ],
+    "retry": {
+      "strategy": "exponentialBackoff",
+      "maxRetryCount": 5,
+      "minimumInterval": "00:00:05",
+      "maximumInterval": "00:30:00"
+    },
+    "scriptFile": "../dist/AnalyticsMessageStatusFromQueue/index.js"
```

Comment:
ditto. Also a typo here `AnalyticsMessageStatusStorageQueueInbloundAdapter` instead of `AnalyticsMessageStatusStorageQueueInboundAdapter`

Reply:
fixed